### ISSUE TITLE
Would it be possible to allow watermarks to be organized in sub-folders of the watermarks folder.....

### ISSUE THREAD

**Issue by todd-prior** [darktable-org/darktable#14338]
I'm just wondering if it would be possible to use the same file selection approach for watermarks that is used with Lut's, ie allow the use of subfolders in the base folder

In the LUT module you can select a subfolder and then also scroll throught the files in that folder from the drop down.

I have quite a few files that I would like to add to the watermarks folder but organized in subfolders. For now only files in the base watermarks folder are available for selection and you cannot access any of them in the module it they are in a subfolder...

So basically the request is to allow access and selection just as it is now in the LUT's module....

**Comment by difrkaguilar** [comment]
This request https://github.com/darktable-org/darktable/issues/9955 is similar to that you like, I post a request to open other folder than the chosen one for the watermarks

Here is a full threat related to open in other folder, with a different purpose (color palettes). https://github.com/darktable-org/darktable/pull/9991

**Comment by todd-prior** [comment]
I find I do sky swaps sometimes by masking this module. I have a large-ish... collection of skies... It would be nice to be able to organize and select them... but I guess there are some concerns about broken edits if things are renamed down the line...I guess I don't see how this is much different than if you mess with your LUT's... so I guess I would lobby still to have the same ability to select nested folders as you can in the 3DLUT module...

**Comment by Mark-64** [comment]
Some time ago I proposed a PR with this feature (#9991) but it didn't get love.
